README.md

```diff
@@ -49,10 +49,11 @@ The JSON file is a map lookup for lowercased mime types.
 Each mime type has the following properties:
 
 - `.source` - where the mime type is defined.
-    If not set, it's probably a custom media type.
     - `apache` - [Apache common media types](http://svn.apache.org/repos/asf/httpd/httpd/trunk/docs/conf/mime.types)
     - `iana` - [IANA-defined media types](http://www.iana.org/assignments/media-types/media-types.xhtml)
     - `nginx` - [nginx media types](http://hg.nginx.org/nginx/raw-file/default/conf/mime.types)
+    - `mime-support` [Debian Linux mime.types database (upstream source)](https://salsa.debian.org/debian/mime-support/-/raw/master/mime.types)
+    - `custom` - Loaded from custom set of types maintained in this repository.
```

Comment:
Please revert this inside this PR and if you think calling this out is worthwhile, please open a PR or issue specifically about this. We don't currently because the JSON file is already quite large, so keeping one of the source to blank really helps folks who are sending this JSON to web clients.

Reply:
OK. Done.